src/emulator/config/include/config/config.h

```diff
@@ -53,5 +53,5 @@ struct Config {
     bool load_config = false;
     bool discord_rich_presence = true;
     optional<bool> wait_for_debugger;
-
+    std::string online_id = "Vita3K";
```

Comment:
fail rebase i supose :P, news have one line space ^^

Reply:
I don't think there should be a newline here.

Reply:
you not see one line removed ? i speek on master ^^
and so like you see on my pr, i keep this news line
![image](https://user-images.githubusercontent.com/5261759/62014046-19f4d100-b19c-11e9-8bcc-a7fd91cd03bb.png)

Reply:
Either way it is not a big deal. In my personal projects, I never leave a new line at the end of a class/struct. This seems to be the case with `HostState` and other definitions.

Reply:
i think its leftover from rebase. let me remove

Reply:
i thought on here it has one newline. on master it's probably rebase fault, so lets ignore

Reply:
not sur, i see here, all get have one news line before ending struct ^^